docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -379,6 +457,18 @@ Serialization Formats
         The public key format used by OpenSSH (e.g. as found in
         ``~/.ssh/id_rsa.pub`` or ``~/.ssh/authorized_keys``).
 
+.. class:: ParameterFormat
+
+    .. versionadded:: 1.9
+
+    An enumeration for parameters formats. Used with the ``parameter_bytes``
+    method available on
+    :class:`~cryptography.hazmat.primitives.asymmetric.dh.DHParametersWithSerialization`.
+
+    .. attribute:: ASN1
+
+        Abstract Syntax Notation One.
```

Comment:
You can inspect the output via an asn1 parser and then hunt around to see if this is documented in some fashion. If this serialization is interoperable with other crypto implementations then there's a name for it (even if it's only a de facto standard).